.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Here are some checklists you may like to use. Use your judgement.
+
+If you're new to the codebase, or you have questions, CONTRIBUTING has many answers.
+-->
+Pre-submit checklist:
```

Comment:
I'd add, "Aim for one logical change per commit."

Ideally, reverting a commit should roll back _one_ change. Not half of one. Not several. A commit message that says, "Did X _and_ Y _and_ Z." is a sign you've mixed commits.

It's not always possible or practical, but it's a good aim.

Reply:
Ah, I see you've addressed this later on. In that case, I'm unsure whether it's worth having a point in the checklist.

🤔 

I think it is. But I'm not wedded to it. :-)